import-export-cli/integration/testutils/testConstants.go

```diff
@@ -98,3 +98,30 @@ const GraphQLEndpoint = "http://www.mocky.io/v2/5ea84def2d0000a52d3a3ecd"
 
 // Web Socket API Endpoint URL
 const WebSocketEndpoint = "ws://echo.websocket.org:80"
+
+// APIHttpRestEndpointWithoutLoadBalancingOrFailoverParamsFile : HTTP/REST Endpoint without Loadbalancing and Failover URLs in api_params.yaml
+const APIHttpRestEndpointWithoutLoadBalancingOrFailoverParamsFile = "testdata/EnvParamsFiles/api_params_http_rest_endpoint_without_lb_or_failover.yaml"
```

Comment:
Since you have moved the params file into another level (/EnvParamsFiles), shall we introduce a separate constant as 
**EnvParamsFilesDir = "testdata/EnvParamsFiles"** or something like this and use it in all the places, to avoid mistakes that can happen when maintaining the code?